### PR TITLE
Grant homepage events

### PR DESCRIPTION
**Pull request by Grant-Holloway**
This adds the functionality for listing Events on the homepage !

### CODE REVIEW

File: KYHBPA_TeamA/KYHBPA_TeamA/App_Start/BundleConfig.cs [hunk 1]
```diff
@@ -14,6 +14,12 @@ public static void RegisterBundles(BundleCollection bundles)
             bundles.Add(new ScriptBundle("~/bundles/jqueryval").Include(
                         "~/Scripts/jquery.validate*"));
 
+            bundles.Add(new ScriptBundle("~/Content/fontawesome").Include(
+                    "~/Content/fontawesome.js",
+                    "~/Content/solid.js",
+                    "~/Content/regular.js"
+                ));
+
```
Comment (tbrownie98) [inline KYHBPA_TeamA/KYHBPA_TeamA/App_Start/BundleConfig.cs:22]: What's this for? @Grant-Holloway

File: KYHBPA_TeamA/KYHBPA_TeamA/Content/Site.css [hunk 2]
```diff
@@ -21,54 +21,114 @@
...
 }
 
 .img-gallery-img{
     width:100%;
     margin:auto;
 }
 
-.carousel-inner > .item > img{
-    margin:auto;
-    max-height:700px;
```
Comment (tbrownie98) [inline KYHBPA_TeamA/KYHBPA_TeamA/Content/Site.css:66]: Why did u do this? @Grant-Holloway

File: KYHBPA_TeamA/KYHBPA_TeamA/Content/Site.css [hunk 3]
```diff
@@ -21,54 +21,114 @@
...
+    letter-spacing: .10em;
+}
+
+.carousel-inner > .item > img {
+    margin: auto;
 }
 
 .carousel{
-    max-height:700px;
-    min-height:700px;
```
Comment (tbrownie98) [inline KYHBPA_TeamA/KYHBPA_TeamA/Content/Site.css:71]: And this? @Grant-Holloway

**Comment by pmgreg3** [comment]
it functions! could you rebase the branch onto master and resolve the conflicts so we can push to master.  there is a space on the front page for the events.

**Pull request closed without merging**